homework/index.js

```diff
@@ -0,0 +1,30 @@
+const promise = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+  .then(response => response.json());
+
+function selectRepo() {
+  promise.then((data) => {
+    data.forEach((i) => {
+      const repolist = document.createElement('option');
+      const opt = document.createTextNode(i.name);
```

Comment:
An `<option>` element has some nice shortcuts: you can just set its `value` and `text` properties.